docs/tutorial/deploy.md

```diff
@@ -3,147 +3,183 @@
 
 This is a part of the [Charmed Apache Kafka Tutorial](index.md).
 
-## Deploy Charmed Apache Kafka (and Charmed ZooKeeper)
+## Deploy Charmed Apache Kafka
 
-To deploy Charmed Apache Kafka, all you need to do is run the following commands, which will automatically fetch [Apache Kafka](https://charmhub.io/kafka?channel=3/stable) and [Apache ZooKeeper](https://charmhub.io/zookeeper?channel=3/stable) charms from [Charmhub](https://charmhub.io/) and deploy them to your model. For example, to deploy a cluster of five Apache ZooKeeper units and three Apache Kafka units, you can simply run:
+To deploy Charmed Apache Kafka, all you need to do is run the following commands, which will automatically fetch [Apache Kafka](https://charmhub.io/kafka?channel=4/edge) from [Charmhub](https://charmhub.io/) and deploy it to your model.
+
+For example, to deploy a cluster of three Apache Kafka brokers, you can simply run:
+
+```shell
+$ juju deploy kafka -n 3 --channel 4/edge --roles=broker
+```
+
+Apache Kafka also uses the KRaft consensus protocol for coordinating broker information, topic + partition metadata and Access Control Lists (ACLs), ran as a quorum of controller nodes using the Raft consensus algorithm.
+
+```{note}
+KRaft replaces the dependency on Apache ZooKeeper for metadata management. For more information on the differences between the two solutions, please refer to the [upstream Apache Kafka documentation](https://kafka.apache.org/40/documentation/zk2kraft.html)
+```
+
+Charmed Apache Kafka can run both with `roles=broker` and/or `roles=controller`. To deploy a cluster of three KRaft controllers, run:
 
 ```shell
-$ juju deploy zookeeper -n 5
-$ juju deploy kafka -n 3
+$ juju deploy kafka -n 3 --channel 4/edge --roles=controller controller
 ```
 
-After this, it is necessary to connect them:
+After this, it is necessary to connect the two clusters, taking care to specify which cluster is the orchestrator:
 
 ```shell
-$ juju relate kafka zookeeper
+juju integrate kafka:peer-cluster-orchestrator kraft:peer-cluster
 ```
 
-Juju will now fetch Charmed Apache Kafka and Charmed Apache Zookeeper and begin deploying them to the LXD cloud. This process can take several minutes depending on how provisioned (RAM, CPU, etc) your machine is. You can track the progress by running:
+Juju will now fetch Charmed Apache Kafka and begin deploying both applications to the LXD cloud before connecting them to exchange access credentials and machine endpoints. This process can take several minutes depending on the resources available on your machine. You can track the progress by running:
 
 ```shell
 juju status --watch 1s
```

Comment:
chore: We need to update to the watch command, instead of `--watch` parameter

Reply:
It's not deprecated yet is it?

Reply:
It is not deprecated, at least yet, but someone said there is a memory leak when we use it with `--watch`. So the recommended approach is to use the watch command instead.

Reply:
Actually I'll update them all.